modules/load-balancer-target-group/main.tf

```diff
@@ -34,17 +43,23 @@ resource "aws_alb_target_group" "tg" {
 # ---------------------------------------------------------------------------------------------------------------------
 
 resource "aws_alb_listener_rule" "http_path" {
-  count = "${var.num_listener_arns}"
+  count = var.num_listener_arns
 
-  listener_arn = "${element(var.listener_arns, count.index)}"
-  priority     = "${var.listener_rule_starting_priority + count.index}"
+  listener_arn = element(var.listener_arns, count.index)
+  priority     = var.listener_rule_starting_priority + count.index
 
   action {
-    target_group_arn = "${aws_alb_target_group.tg.arn}"
+    target_group_arn = aws_alb_target_group.tg.arn
     type             = "forward"
   }
 
-  condition = "${var.routing_condition}"
+  dynamic "condition" {
+    for_each = var.routing_condition
+    content {
+      field  = lookup(condition.value, "field")
+      values = lookup(condition.value, "values")
```

Comment:
Should these be `condition.value["field"]` and `condition.value["value"]`? Omitting default in `lookup` is deprecated according to https://www.terraform.io/docs/configuration/functions/lookup.html

Reply:
Ah, didn't realize it was deprecated. Updated: https://github.com/gruntwork-io/terraform-aws-couchbase/pull/49/commits/30031d392201098a64abb8b68ac840b239593841